style.css

```diff
@@ -2,3 +2,218 @@
+    float: right;
+    z-index: 4;
+    display: flex;
+    border-right: 2px solid black;
+    margin-right: 15px;
+}
+.cart {
+    background-color: white;
+    width: 25px;
+    height: 25px;
+}
+.header_cart img {
+    width: 25px;
+    height: 25px;
+    padding-right: 5px;
+    background-color: white;
+}
+.header_cart h5 {
+    padding: 3px 10px 0px 5px;
+}
+#renome {
+    margin-left: 10px;
+    margin-bottom: 10px;
+    font-family: 'Pacifico', cursive;
+    font-weight: 200;
+    color:black;
+    z-index: 5;
+}
+.header_body {
+  position: relative; 
+  display: flex;
+  justify-content: space-between;
+  height: 80px;
+  align-items: center;
+}
+.container_menu {
+  position: relative; 
+  display: flex;
+}
+.header_burger {
+    display: none;
+}
+.header_menu {
+ 
+}
+.header_list {
+    display: flex;  
+    position: relative;
+    z-index: 2;
+    padding: 0px 15px 0px 0px;
+}
+.cart_list {
+    display: none;
+}
+.cart_list.active {
+    display: block;  
+    position: relative;
+    right: 50px;
+    z-index: 2;
+    padding: 20px 10px 10px 10px;
+    width: 100px;
+    height: 140px;
+    background-color: white;
+}
+.header_list li {
+    list-style: none;
+    margin: 0px 0px 0px 10px;
+}
+.cart_list li {
+    list-style: none;
+    margin: 10px 10px 10px 10px;
+    text-align: center;
+}
+.header_link {
+   text-transform: uppercase;
+   font-size: 12px;
+   text-decoration: none;
+   color: black;
+   font-family: 'Montserrat', sans-serif;
+   font-weight:bolder;
+}
+.cart_link {
+    font-size: 12px;
+    text-decoration: none;
+    color: black;
+    font-family: 'Poppins', sans-serif;;
+    font-weight:bolder;
+    text-align: center;
+ }
+.content {
+    padding: 100px 0px 0px 0px;
+}
+@media (max-width: 767px){
+    .header_body {
+        height: 50px;
+    }
+    .header_logo {
+        flex: 0 0 40px;
+    }
+    .header_burger {
+        display: block;
+        position: relative;
+        width: 30px;
+        height: 20px;
+        z-index: 3;
+    }
+    .header_burger:before,
+    .header_burger:after {
+        content: '';
+        position: absolute;
+        width: 100%;
+        height: 2px;
+        left: 0;
+        background-color: black;
+    }
+    .header_burger:before {
+        top: 0;
+    }
+    .header_burger:after {
+        bottom: 0;
+    }
+    .header_burger span {
+        position: absolute;
+        width: 100%;
+        height: 2px;
+        left: 0;
+        top: 9px;
+        background-color: black;
+        transition: all 0.3s ease 0s;
+    }
+    .header_burger.active:before {
+        transform: rotate(45deg);
+        top: 9px;
+    }
+    .header_burger.active:after {
+        transform: rotate(-45deg);
+        bottom: 9px;
+    }
+    .header_burger.active span {
+        transform: scale(0);
+    }
+    .header_menu {
+        position: fixed;
+        top: -100%;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        padding: 70px 10px 20px 10px;
```

Comment:
Can be shortened to : padding: 70px 10px 20px;

Reply:
> Can be shortened to : padding: 70px 10px 20px;

fixed